packages/shared/src/components/streak/popup/ReadingStreakPopup.tsx

```diff
@@ -79,13 +85,15 @@ export function ReadingStreakPopup({
   streak,
   fullWidth,
 }: ReadingStreakPopupProps): ReactElement {
+  const isTablet = useViewSize(ViewSize.Tablet);
```

Comment:
I see we're using this to differentiate whether it's mobile or other devices, rather then just tablet. I'd be more in favour of using mobile check as it's a bit missleading.

Reply:
Changed it. Think I intended to use it for something else, since I used isTablet 😬